environments/staging-aws-test/public.yml

```diff
@@ -40,10 +40,8 @@ s3_blob_db_s3_bucket: 'dimagi-commcare-staging-test-blobdb'
 
 localsettings:
   ALLOWED_HOSTS:
-    - staging.commcarehq.org
```

Comment:
Why does this one also get removed?

Reply:
It could have been removed before this PR as well—this isn't the actual value of SITE_HOST for this environment, so it was unnecessary (but harmless) to include (and SITE_HOST is automatically included, so that's uncessary to include too).